udata_transport/templates/dataset-transport.html

```diff
@@ -0,0 +1,8 @@
+{%- if transport_url %}
+<div class="bg-alt-blue-cumulus fr-p-3v fr-mb-1w">
+    <span class="fr-grid-row">
+      <span class="fr-fi-external-link-line fr-mr-1w" aria-hidden="true"></span>
+      Ce jeu de données est aussi référencé sur le portail transport.data.gouv.fr <a href={{transport_url}}>Voir.</a>
```

Comment:
How about? (@abulte, let me know if you have a better wording)
```suggestion
      Ce jeu de données est aussi référencé sur le portail transport.data.gouv.fr.&nbsp;<a href={{transport_url}}>Voir ce jeu de données sur le portail transport.</a>
```